### PR TITLE
Updated PushIO AAR, firebase and Bluedot SDK to latest versions

### PR DESCRIPTION
**Pull request by nehabluedot**
1. Add `pushio_config.json` from https://interact5.responsys.net/mobileappconsole/c/mobileappconsolemanager/#!mac
2. Set Program `test 25022021` to running.
3. Now wait for zone entry on the device and after a while FCM push should be delivered

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -26,12 +26,14 @@ android {
...
+    implementation fileTree(dir: 'libs', include: ['PushIOManager-6.51.aar'])
+    implementation 'androidx.appcompat:appcompat:1.4.1'
     implementation 'com.android.support.constraint:constraint-layout:2.0.4'
-    implementation 'com.gitlab.bluedotio.android:point_sdk_android:15.3.1'
-    implementation 'com.google.firebase:firebase-core:18.0.2'
-    implementation 'com.google.firebase:firebase-messaging:21.0.1'
+    implementation 'com.gitlab.bluedotio.android:point_sdk_android:15.4.0'
+    implementation 'com.google.firebase:firebase-core:20.1.0'
+    implementation 'com.google.firebase:firebase-messaging:23.0.0'
+    implementation 'com.google.android.gms:play-services-base:18.0.1'
```
Comment (jsprincep) [inline app/build.gradle:35]: Why do we need play services now?

File: app/build.gradle [hunk 2]
```diff
@@ -26,12 +26,14 @@ android {
...
+    implementation 'androidx.core:core:1.7.0'
```
Comment (jsprincep) [inline app/build.gradle:36]: What has changed that we need androidx core now?

File: app/build.gradle (hunk 1 above)
Comment (nehabluedot) [reply app/build.gradle:35]: Play services is recommended by PushIO [here](https://docs.oracle.com/en/cloud/saas/marketing/responsys-develop-mobile/android/step-by-step.htm)
Its recommended for using firebase-messaging library version 21. x.x or later

File: app/build.gradle (hunk 2 above)
Comment (nehabluedot) [reply app/build.gradle:36]: Its added based on the step-by-step guideline shared by [Responsys](https://docs.oracle.com/en/cloud/saas/marketing/responsys-develop-mobile/android/step-by-step.htm)

Comment (jsprincep) [reply app/build.gradle:36]: Ahh, of course, the sdk is an aar, so they cannot package their own dependencies. 😞

**Review by jsprincep — APPROVED** [review]

**Pull request merged**